roles/splunk/vars/main.yml

```diff
@@ -2,6 +2,7 @@
 # Automatically determine desired splunk version string
 splunk_file: "{{ splunk_package_url | regex_search('(splunk(?:forwarder)?\\-\\d+[^/]*\\.tgz)') }}"
 splunk_v: "{{ splunk_package_url | regex_search('(\\d+\\.\\d+\\.\\d+(?:\\.\\d+)?)') }}"
+splunk_v_num_only_url: "{{ splunk_package_url | regex_search('(\\d+\\.\\d+\\.\\d+)') }}"
```

Comment:
You're running the splunk version command to get the current version, so there's no need to also get the version from the url.

Reply:
Oh yeah - you're right. I just included that there in case people were running check_splunk.yml at first. Don't know what I was thinking - I'll remove that.